### DISCUSSION TITLE
Bug Report: custom songs

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#545] [Q&A]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.
windows 86_64/edge
## What version of the game are you using? Look in the bottom left corner of the main menu.
1.4.2/0.2.7.1
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
1. Import custom song
2. Go to debug menu or freeplay/story mode
3. Select song
## Please describe your issue. Provide extensive detail and images if possible.
when I import a custom song it crashes, she song name in the jsons are correct and i have the songs in assets\songs

**Comment by puyoxyz** [comment]
errors in console?

**Reply by ghost** [reply]
<img width="960" alt="funierror" src="https://user-images.githubusercontent.com/78268270/119229330-7c436200-bae5-11eb-9842-c4ee5dca3496.PNG">

**Reply by ghost** [reply]
Add ` -debug` to your command and try again.
